marked-element.html

```diff
@@ -75,7 +75,7 @@
         display: none !important;
```

Comment:
I meant that 'host` has a `display:block` style, which has higher precedence than the user agent stylesheet `'hidden`. As a result, if you don't provide a style for `:host('hidden')`, then `<marked-element hidden>` won't actually hide the element.

This `hidden` however doesn't apply to the host, but to the `.content` div -- a similar argument can be made there, only I don't think anything sets the `display` of that element to anything else. In any case, doesn't hurt to have it